image/templates/helm/stackrox-secured-cluster/.helmtplignore.htpl

```diff
@@ -40,3 +40,4 @@ templates/02-scanner-03-tls-secret.yaml
 templates/02-scanner-v4-03-db-tls-secret.yaml
 templates/02-scanner-v4-03-indexer-tls-secret.yaml
 templates/02-scanner-v4-03-matcher-tls-secret.yaml
```

Comment:
nit: can remove this line. Would also need to update the comment above this talking about TLS secrets

Reply:
cleaned up, thanks!